### PR TITLE
SEO-174775 Xamarin Redirect Links

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have changed Xamarin Page with Redirect links.

Regards,
Yvone.

### CODE REVIEW

File: Xamarin/Calendar/Calendar-MVVM.md [hunk 1]
```diff
@@ -331,4 +331,4 @@ public class CalendarViewModel
 
 ## See also
 
-[How to bind SelectedDates of Calendar in MVVM?](https://www.syncfusion.com/kb/10087/how-to-bind-selecteddates-of-calendar-in-mvvm)
+[How to bind SelectedDates of Calendar in MVVM?](https://support.syncfusion.com/kb/article/8866)
```
Comment (MallikaRK) [inline Xamarin/Calendar/Calendar-MVVM.md:334]: KB link seems in short form. Can you update full URL here?
https://support.syncfusion.com/kb/article/8866/how-to-bind-selecteddates-of-calendar-in-mvvm

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Calendar/Calendar-MVVM.md (hunk 1 above)
Comment (Yvone-Atieno) [reply Xamarin/Calendar/Calendar-MVVM.md:334]: updated

File: Xamarin/DataGrid/Conditional-Styles.md [hunk 2]
```diff
@@ -290,5 +290,5 @@ N> You can refer to our [Xamarin DataGrid](https://www.syncfusion.com/xamarin-ui
 
 ## See also
 
-[How to apply the row background color when mouse hover on SfDataGrid in UWP](https://www.syncfusion.com/kb/11230)
+[How to apply the row background color when mouse hover on SfDataGrid in UWP](https://support.syncfusion.com/kb/article/9779)
```
[the remark this answers is not in the document]
Comment (Yvone-Atieno) [reply Xamarin/DataGrid/Conditional-Styles.md]: updated

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**